roadie-kubernetes-cluster-access/templates/broker.yaml

```diff
@@ -3,25 +3,25 @@
 apiVersion: v1
 kind: ServiceAccount
 metadata:
-  name: {{ .Values.broker.serviceAccount.name }}
+  name: {{ .Values.broker.serviceAccount.name | default (printf "%s-service-account" .Release.Name) }}
```

Comment:
By using the release name in the resource names, the resources can be unique even if they are being deployed to the default namespace.